app/src/main/java/com/github/multimatum_team/multimatum/activity/AddDeadlineActivity.kt

```diff
@@ -36,6 +43,8 @@ class AddDeadlineActivity : AppCompatActivity() {
     private val deadlineListViewModel: DeadlineListViewModel by viewModels()
     private lateinit var textDate: TextView
     private lateinit var textTime: TextView
+    private lateinit var pdfTextView: TextView
+    private var pdfUri: Uri = Uri.EMPTY
```

Comment:
For clarity, it would be good to reorder vals and vars in this file. It means put lateinit together, and val/var by type.